### ISSUE TITLE
Help installing

### ISSUE THREAD

**Issue by derekthecool** [antistic/plover_markdown_dictionary#1]
Hi, this is a really great project.

I really love mark down and I want to use your plugin. But I am having trouble installing.
I am very familiar with using git, but I don't know about using the command `plover`.

Does that require building from source? I am using the latest release as of now Plover 4.0.0.dev10+120.g4394ef1 for windows.

Any tips would be appreciated. I'll will help contribute if I find any bugs as well.

Thanks!

**Comment by antistic** [comment]
Hey

It does not require building from source. `plover` is the same command you use to start it, as [described in the read the docs](https://plover.readthedocs.io/en/latest/cli_reference.html), likely at `C:\Program Files (x86)\Open Steno Project\Plover 4.0.0\plover_console.exe` if you are on windows.

**Comment by derekthecool** [comment]
Thanks antistic,

I mistakenly tried the non console application. That worked to get it installed!

Excellent project!



One note I just remembered. For some reason I was unable to get the install to work on the version I mentioned.
Technically, it installed, but I was unable to load any md files or create new one's.

I rolled back to the latest stable version of 4.0.0.dev10+82.g2012d4b where it worked just fine.
